controller/src/beerocks/bml/bml_defs.h

```diff
@@ -610,6 +619,32 @@ struct BML_NEIGHBOR_AP {
     uint32_t ap_ChannelUtilization;
 };
 
+struct BML_CLIENT_CONFIG {
+    // 1 for true, 0 for false, -1 for "not configured".
+    int8_t stay_on_initial_radio;
+
+    // 1 for true, 0 for false, -1 for "not configured".
+    int8_t stay_on_selected_device;
+
+    // Type of selected band for the client.
+    int8_t selected_bands;
+};
+
+struct BML_CLIENT : BML_CLIENT_CONFIG {
+    // Client MAC.
+    char sta_mac[BML_MAC_ADDR_LEN];
+
+    // Time of last client update (in Seconds)
```

Comment:
what is an "update"? is it a client connection? is it statics-update? need to be specific here:
```suggestion
    // Time of last client configuration edit (in Seconds)
```

Reply:
Changed